schema/defs.yaml

```diff
@@ -66,6 +66,7 @@ description: Common schema definitions shared by all themes
         that referenced property will be specified in the overture list of
         approved sources from the Overture Data Working Group that contains
         the relevant metadata for that dataset including license source organization.
+        The recordid refers to the specific record within the dataset that was used.
```

Comment:
It's IMO preferable to move field-specific documentation into the field itself. A `description` property could be added into the `record[I]d` property.